source/create.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
+module.exports = () => {};
```

Comment:
?

Reply:
Oh, I was playing with cyclic modules. Removed it.